Removing div.usa-grid-full from navigation. closes #65

webinista opened the pull request:
- Also positions logo closer to the left edge of viewport.

modified:   src/_includes/home-header.html
modified:   src/_includes/internal-header.html
modified:   src/_sass/_va-buttons.scss
modified:   src/_sass/_va-rr-header.scss

Commit:
use default top-level as the source directory

Commit:
exclude source files from webserver, set baseurl

Commit:
remove _config.yml from excludes list

Commit:
add trailing slash to site url

Commit:
update excludes

Commit:
Fix typo and add Oxford comma

Commit:
Fix typo in footer

webinista committed:
Removing div.usa-grid-full from navigation. closes #65

- Also positions logo closer to the left edge of viewport.

modified:   src/_includes/home-header.html
modified:   src/_includes/internal-header.html
modified:   src/_sass/_va-buttons.scss
modified:   src/_sass/_va-rr-header.scss

webinista committed:
Removing div.usa-grid-full from navigation. closes #65

- Also positions logo closer to the left edge of viewport.

modified:   src/_includes/home-header.html
modified:   src/_includes/internal-header.html
modified:   src/_sass/_va-buttons.scss
modified:   src/_sass/_va-rr-header.scss

webinista committed:
Removing div.usa-grid-full from navigation. closes #65

- Also positions logo closer to the left edge of viewport.

modified:   src/_includes/home-header.html
modified:   src/_includes/internal-header.html
modified:   src/_sass/_va-buttons.scss
modified:   src/_sass/_va-rr-header.scss

webinista committed:
Merge pull request #85 from department-of-veterans-affairs/footer-typo

Fix grammatical error in footer.

webinista committed:
Merge pull request #84 from department-of-veterans-affairs/index-typo-and-oxford-comma

Fix typo and add Oxford comma

Commit:
exclude vendor

Commit:
put site in /roadrunner subdirectory

Commit:
fix footer link for htmlproof, and add some new rake targets

Commit:
moar better footer link

Commit:
url in config file should not include baseurl

emilyville commented:
Code looks good, but viewing locally is a little close to the tabs
![screen shot 2016-02-29 at 10 58 57 pm](https://cloud.githubusercontent.com/assets/1307774/13420093/0cc07e7e-df38-11e5-9300-447f0d99f2dd.png)

If we want to get to tabs in another PR, that's fine, too.

Commit:
Remove copyright notice

awong-dev committed:
Merge pull request #83 from department-of-veterans-affairs/gh-pages-src

src change - for gh-pages

webinista committed:
Merge pull request #92 from department-of-veterans-affairs/plusjeff-patch-1

Remove copyright notice

webinista committed:
Fixing merge by adding _includes/home-header.html

Commit:
Only build master branch

webinista committed:
Removing src directory since the restructuring.

webinista committed:
Removing src directory since the restructuring.

Commit:
Merge pull request #94 from department-of-veterans-affairs/travis-builds

Only build master branch

webinista committed:
Top navigation adjustments

- Increasing margin around top navigation
- Hiding navigation at narrower viewports
- Moving _sass/_va-rr-header.scss to components directory

webinista committed:
Merge branch 'issue#65' of github.com:department-of-veterans-affairs/roadrunner into issue#65

webinista committed:
Forgot to add this update to previous commit.

webinista commented:
Updated the CSS so that the tabs are further from the logo and also hidden on mobile.

emilyville commented:
:+1:
